### PR TITLE
Questions.jsx and Quotes.jsx mobile responsiveness fixes

### PR DESCRIPTION
**Pull request by swedenagu**
Screenshot 2024-01-23 at 7 16 44 PM
Screenshot 2024-01-23 at 7 17 19 PM
Screenshot 2024-01-23 at 7 17 56 PM

### CODE REVIEW

**Review by ysh0305 — CHANGES_REQUESTED** [review]
The website's original view is changed. You should not be changing the original formatting. You can put md: to keep the original setting on the laptop view and change only the mobile view.  The mobile still need to be improved

**Comment by swedenagu** [comment]
> The website's original view is changed. You should not be changing the original formatting. You can put md: to keep the original setting on the laptop view and change only the mobile view. The mobile still need to be improved

Screenshot 2024-01-25 at 6 53 57 PM
Screenshot 2024-01-25 at 6 55 10 PM
Screenshot 2024-01-25 at 6 55 38 PM
Screenshot 2024-01-25 at 6 55 55 PM

File: src/components/join/Questions.jsx [hunk 1]
```diff
@@ -2,16 +2,22 @@ import React from "react";
...
           <div
-            className=" border-l-robosub-blue w-1/4 border-l-8 pl-[4%] font-bold"
+            className=" border-l-robosub-blue text-sm lg:w-1/4 border-l-8 pl-[4%] md:pr-[0%] pr-[8%] font-bold "
             key={index}
           >
             {item.question}
           </div>
-          <div className="w-2/3 mr-[7%] text-sm" key={index}>
+          <div
+            className="w-2/3 md:max-w-full mr-[7%] text-xs pr-2 md:pr-8"
```
Comment (ysh0305) [inline src/components/join/Questions.jsx:18]: Formatting for the laptop view are changed

File: src/components/join/Quotes.jsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { quotes, quotes2 } from "@/data/quotes";
 import Quote from "./Quote";
 const Quotes = () => {
   return (
-    <div className="grid grid-cols-2 mt-[3%] w-2/3">
+    <div className="grid lg:grid-cols-2 mt-[3%] w-2/3 md:max-w-full text-xs">
```
Comment (ysh0305) [inline src/components/join/Quotes.jsx:5]: Formatting for the laptop view are changed

File: src/components/join/Questions.jsx [hunk 3]
```diff
@@ -2,16 +2,22 @@ import React from "react";
...
+    <div className="mt-3 w-3/4">
       {questions.map((item, index) => (
-        <div key={index} className="flex ml-[7%] justify-between pb-[4%]">
+        <div
+          key={index}
+          className="flex ml-[7%] justify-between pb-[4%] md:mr-[0%] mr-[-8%]"
+        >
           <div
-            className=" border-l-robosub-blue w-1/4 border-l-8 pl-[4%] font-bold"
+            className=" border-l-robosub-blue text-sm lg:w-1/4 border-l-8 pl-[4%] md:pr-[0%] pr-[8%] font-bold "
```
Comment (ysh0305) [inline src/components/join/Questions.jsx:12]: Formatting for the laptop view are changed

**Review by ysh0305 — CHANGES_REQUESTED** [review]
Overall, there are quite a few formatting on the laptop view that shouldn't be changed as you should only be changing the mobile view.  I think it is easier for you to discard your changes and restart the issues with a new branch (restart from step 1). If your local repo is not updated to the new dev branch on GitHub, you can reclone the repo and delete the old folder.

**Pull request closed without merging**